### PR TITLE
Decouple manage ws vcsrepo resource

### PR DESCRIPTION
**Pull request by Spredzy**
This pull request target two improvement : 

 * Decoupling of `$manage_ws` and the `vcsrepo` resource
 * Improving flexibility of the `apache::vhost` resource

### CODE REVIEW

**Comment by Spredzy** [comment]
Up? :)

**Comment by thejandroman** [comment]
Sorry, haven't had a chance to review this yet. I'll check it out today.

File: manifests/init.pp [hunk 1]
```diff
@@ -102,6 +107,7 @@
   $ws_servername    = $::kibana3::params::ws_servername,
   $ws_port          = $::kibana3::params::ws_port,
   $ws_default_vhost = $::kibana3::params::ws_default_vhost,
+  $ws_extras        = {},
```
Comment (thejandroman) [inline manifests/init.pp]: Let's do input validation on this new param.

**Comment by Spredzy** [comment]
@thejandroman updated.

**Comment by thejandroman** [comment]
Thanks @Spredzy

**Pull request merged**